node/metrics/auth.ts

```diff
@@ -3,19 +3,22 @@ import type { Logger } from '@vtex/api/lib/service/logger/logger'
 import type { Metric } from '../clients/metrics'
 import { B2B_METRIC_NAME, sendMetric } from '../clients/metrics'
 
-export interface AuthAuditMetric {
+export interface AuthAuditTokenMetrics {
```

Comment:
do we really need to break these into 2 interfaces? It seems to be adding some unnecessary complexity.

Reply:
Done